### ISSUE TITLE
anyone knows how to get hogwarts legacy to work?

### ISSUE THREAD

**Issue by AmSoFancy** [Nukem9/dlssg-to-fsr3#105]
I tried both methods and the window never pops out, although it worked with other games. Does anyone know how to fix it?

**Comment by ejams1** [comment]
I just tried it with 0.5 and it works via the DLSSTweaks method. Copy the files to
```
Hogwarts Legacy\Phoenix\Binaries\Win64
```
and make the change in the dlsstweaks.ini as specified in the README included in the download from this repo so that DLSSTweaks points to this FSR3 wrapper

**Comment by ejams1** [comment]
Also when you enable DLSS to enable frame gen, don't touch the setting - leave it on automatic. It seems sensitive to change and will start crashing if you alter it using 0.5.

If you do happen to change it and end up in a crash loop, you just need to disable this mod by moving the files back out of the binary directory or comment out the ini configuration line again, and go back in game to disable DLSS to try again.

**Comment by AmSoFancy** [comment]
thanks for the reply, I tried again but no avail. It's weird cuz it worked on other games.

**Comment by Aidorin0** [comment]
Probably the cause of Hogwarts legacy not working for some people is because they are using the Jack sparrow version.

**Comment by ejams1** [comment]
> Probably the cause of Hogwarts legacy not working for some people is because they are using the Jack sparrow version.

It's also an unreal engine game which has a shortcut exe which is confusingly where the game exe would normally be in other games, so if you don't know that then it simply won't work

**Comment by ngrfgt** [comment]
I have the Jack Sparrow version and this doesn't work.

**Comment by Renfel7** [comment]
Crashes if DLSS is enabled before launching or if enabling Frame Gen / DLSS after launch. I have installed DLSS v3.5 though so this could be the cause. What is the current DLSS version Hogwarts uses (non-modded)? I can get that, give it a try and report back.

**Comment by Aidorin0** [comment]
> > Probably the cause of Hogwarts legacy not working for some people is because they are using the Jack sparrow version.
> 
> It's also an unreal engine game which has a shortcut exe which is confusingly where the game exe would normally be in other games, so if you don't know that then it simply won't work

Yeah, but a lot of people are placing the mod files in the correct path (the one you posted 3 or 4 comments above) and still doesn't work. Probably because of that Jack sparrow thing.

**Comment by ngrfgt** [comment]
![Untitled](https://github.com/Nukem9/dlssg-to-fsr3/assets/27920734/d2874906-b795-4e49-9cfe-be1597cf2ee8)
This is the log.

**Comment by Neegger** [comment]
I use DLSS 3.5.10 with DLSSTweaks set to native rendering.

The game runs with FG and HDR on, jumping from 50 to mid 60s at ultimate with RT high.

Then after 2-3 minutes of actually playing it crashes, doesn't matter if HDR is on or off.

**Comment by slyfox8900** [comment]
Can confirm the "Jack Sparrow" version doesn't work. Possibly because its not using an updated NVidia Streamline SDK, pretty sure the empress version uses the initial launch version of the game. Another game i have that runs on the same engine that is an official copy of the game works fine and its on the most up to date game version. But has the same folder layout as Hogwarts Legacy. 

Hogwarts does work if its from steam or any other official platform.

**Comment by Neegger** [comment]
> Hogwarts does work if its from steam or any other official platform.

Incorrect, I have it on Steam and it randomly crashes

**Comment by slyfox8900** [comment]
> > Hogwarts does work if its from steam or any other official platform.
> 
> Incorrect, I have it on Steam and it randomly crashes

Sorry, i meant it straight up doesnt hook the dll with the pirated copy. With steam or other platforms it hooks fine. Latest v0.6 might help with crashing. Give it a try

**Comment by Neegger** [comment]
> Sorry, i meant it straight up doesnt hook the dll with the pirated copy. With steam or other platforms it hooks fine. Latest v0.6 might help with crashing. Give it a try

Sadly the problem still stands even with v0.6

**Comment by ZeroGravity411** [comment]
How does one know if they have the "Jacksparrow" version?

**Comment by craxton** [comment]
> How does one know if they have the "Jacksparrow" version?

pirated version/not updated. but, i have a legit paid version can confirm on epic games version hogwarts legacy crashes.
